lib/hls/hls_parser.js

```diff
@@ -2145,6 +2197,8 @@ shaka.hls.HlsParser = class {
         },
     );
 
+    // If AES-128 is used, this option should be enabled to decrypt the full
+    // segment.
     if (this.config_.hls.useFullSegmentsForStartTime) {
```

Comment:
I think that instead of doing this configuration manually, if it is detected that the stream is AES-128, the entire segment should always be requested automatically.

Reply:
Done.